### PR TITLE
Restyle pagination

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Refactors the styling used for the pagination buttons to prefer Source

Along the way, I move the margins from the left to the right so we are flush to the left now

### Before
Screenshot 2020-04-28 at 14 40 04

### After
Screenshot 2020-04-28 at 14 39 24

## Why?
There was an alignment issue with the icon in Firefox so rather than fix that I leant into Source but with overrides

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "@guardian/discussion-rendering",
     "description": "This codebase started as a hack day project by @gtrufitt and @nicl. The purpose is parity of the existing discussion application on Frontend using the discussion API (search for Private Repo).",
-    "version": "2.2.6",
```
Comment (gtrufitt) [inline package.json:4]: Isn't this on master?

Comment (oliverlloyd) [reply package.json]: Keen eyes! I forgot to merge the other PR. Done now and merged into here

**Comment by paperboyo** [comment]
Aside: at least the rhs one doesn't look like it's [optically centred](https://medium.muz.li/optical-effects-9fca82b4cd9a) horizontally.

**Comment by paperboyo** [comment]
Thanks, @oliverlloyd!

**Pull request merged**